src/Features/CSharp/Portable/EditAndContinue/StatementSyntaxComparer.cs

```diff
@@ -503,6 +520,38 @@ internal static Label Classify(SyntaxKind kind, SyntaxNode? node, out bool isLea
                 case SyntaxKind.AwaitExpression:
                     return Label.AwaitExpression;
 
+                case SyntaxKind.TypeParameterList:
+                    isLeaf = false;
```

Comment:
>isLeaf = false; [](start = 20, length = 15)

nit: `isLeaf` is false by default, no need to assign it unless we want to set it to true